scripts/effects/effect-helpers.js

```diff
@@ -140,4 +140,64 @@ export default class EffectHelpers {
 
     return activeEffect.update({ statuses: statusesArray });
   }
+
+  /**
+   * Migrates the description of Custom Convenient Effects to use the ActiveEffect5e#description field instead of the legacy flags.
+   *
+   * @param {string} id - the ID of the custom effects item
+   * @returns {Promise} a promise that resolves when all the Item's ActiveEffects descriptions have migrated
+   */
+  async _migrateCustomEffectsItemEffectDescriptions(id, migration) {
```

Comment:
We probably want to do a ui notification indicating this is taking place

Reply:
Added!